src/IObjectMakr.ts

```diff
@@ -24,7 +24,7 @@ export interface IClassFunctions {
  * Root abstract definition for class Functions.
  */
 export interface IClassFunction {
-    new (): any;
+    (): any;
```

Comment:
the only way I could get `function () { }` in `processFunctions` to not cause any syntax errors was if I changed IClassFunction by cutting off the `new` which in then turn caused me to cut off `new` in other places so there would be no syntax errors (which you noticed and added a comment below this one)